entrypoint.sh

```diff
@@ -2,6 +2,8 @@
 
 set -eo pipefail
 
+cd "$INPUT_WORKING_DIR"
+
 IFS=
 OUTPUT="$(dnscontrol "$1")"
```

Comment:
```suggestion
OUTPUT="$(dnscontrol "$1" --config "$DNSCONTROL_CONFIG_FILE")"
```

Reply:
@koenrh I am for this, but I believe we will also need the ability to pass in the path to the creds file with `--creds` as well. Does that track with what you are thinking?